toolkit/scss/_instruction-list.scss

```diff
@@ -0,0 +1,105 @@
+@import "_colours.scss";
+@import "_typography.scss";
+@import "_measurements.scss";
+@import "_shims.scss";
+@import "_conditionals.scss";
+
+ul,
```

Comment:
I took this from the list scss in the supplier app, but yes you're right, it is, and probably has been redundant for a while.